package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codahq/packs-sdk",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
Per my Slack comment, how do we feel about something like `1.4.1-prelease1`, so we can increment the suffix each time we make a change rather than having to make our actual public semantic versions non-contiguous?

I think the `semver` checks we do in `coda` are ok with this syntax, but obviously needs a little bit of verification.

Reply:
Changed to `1.4.1-prerelease.1`